cartoframes/viz/layer.py

```diff
@@ -19,8 +19,8 @@ class Layer(object):
     notebook to get a preview of a Layer.
 
     Args:
-        source (str, :py:class:`CartoDataFrame <cartoframes.CartoDataFrame>`,
-            pandas.DataFrame, geopandas.GeoDataFrame): The source data.
+        source (str, pandas.DataFrame, geopandas.GeoDataFrame,
+            :py:class:`CartoDataFrame <cartoframes.CartoDataFrame>`): The source data.
```

Comment:
How about explicitly saying "table, query, or three dataframes listed". Some people might not know what `str` could give them

Reply:
OK

Reply:
Since we are using this (...) section for the types, I would improve the definition of the parameter instead.